boot/freeldr/freeldr/arch/i386/entry.S

```diff
@@ -199,6 +202,20 @@ _PxeCallApi:
 
     ret
 
+PUBLIC _HwMouseIrqHandler
+_HwMouseIrqHandler:
```

Comment:
I'm thinking whether it could be better to move this mouse-specific code into a new .S file, much like what is done in arch/i386/drvmap.S
(OK this requires creating a new file, but this may help having more tidy code...)